tests/e2e/send.test.js

```diff
@@ -261,6 +250,17 @@ describe('Send View', () => {
 
     await page.waitForTestIdSelector('contact-name-Subaccount');
   });
+
+  test('choosing contact with icns name from address book and sending icp', async () => {
+    await waitForAmount(page);
+    const previousAmount = await getAvailableAmount(page);
+    await contactSelect(page);
+    await page.waitForTimeout(100);
+    await sendToken(page);
+    await popupPageUtils.refreshWallet(page);
+    await waitForBalanceChange(page);
```

Comment:
Oki, let's try this:
I assuming line `260` makes a reload....if that is so
What if we change `await waitForBalanceChange` to `page.waitForNavigation()` ? 
It seems this last command will wait for everything to be rendered. more info here: `https://devdocs.io/puppeteer/index#pagewaitfortimeoutmilliseconds`

Reply:
A sample of this would be:

    //change the defualt naviagation wait time
    await page.setDefaultNavigationTimeout(10000);

    await popupPageUtils.refreshWallet(page)
    
   **Optional:** One thing I would do there, would be:
assert.(previousAmount).notEqual(getAvailableAmount)

To implement this, some things needs to be done first because previousAmount will call again getAvailableAmount(page) and indeed is going to be the same value, so you will have to push the value into a new variable, and it might look something like this. 

const storePreviousAmount = []

storePreviousAmount.push('locator')

and then 
assert.(storePreviousAmount).notEqual(getAvailableAmount)

This optional, and a general idea, and maybe this is what you are doing on line 262, but I haven't found `tokenBalanceCheck` to fully confirm what's happening there.